Quadriga/src/main/java/edu/asu/spring/quadriga/qstore/impl/QStoreConnector.java

```diff
@@ -466,6 +468,7 @@ public String searchNodesByConcept(String conceptId) throws Exception {
         try {
             // Get complete network xml from QStore
             res = restTemplate.postForObject(getQStoreSearchUrl(), request, String.class);
+            logger.info("Response from QStore", res);
```

Comment:
this does not need to be logged as info. would just clutter the log files. If at all, log it as debug.